1.8/install/database-restore.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: page
+title: "Database Restore"
+categories: [install]
+
+redirect_from:
+- /Database-Restore.html
+---
+
+To restore a database to your forum, you will first need to know what database type your forum is using in order to determine which instructions to follow. MyBB supports the following database types:
```

Comment:
`database type` might make sense to be worded as `database engine`, as I believe that's what we call it in the installer (I think, off the top of my head).